### ISSUE TITLE
exclamation marks on completed quests remain after claiming reward

### ISSUE THREAD

**Issue by GritsanY** [FTBTeam/FTB-Quests#373]
Hello, I'm running FTB Stoneblock 2 server on my dedicated Ubuntu VM.

For example, I'm connecting from my work PC and complete some quests. I get the rewards and the quests are displayed as completed.

But if I connect from my home PC later using the same account, I see all previously completed quests with exclamation marks, as if the reward was not claimed.

I can progress further, meaning all related quests remain unblocked; and I can't get the rewards second time. But it becomes hard to follow the progression status. Does anyone run into this and can somebody help me mitigate it?
![изображение](https://user-images.githubusercontent.com/17610768/73252333-687f3880-41dc-11ea-8f07-2c7bd2f3308e.png)
Here on this picture I have claimed all rewards.

**Comment by gaminggirl** [comment]
After last update this started happening in my pack to

**Comment by MaxNeedsSnacks** [comment]
The Quests version used in Stoneblock 2 is obsolete even for 1.12 standards, there may be updates out for the pack on the FTB App, but otherwise, you'll likely have to live with these exclamation marks for this pack specifically, sorry about that
